src/Symfony/Component/Mailer/Bridge/MicrosoftGraph/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "symfony/microsoft-graph-mailer",
```

Comment:
the factory is named `.microsoftgraph` without underscore, this looks inconsistent, but I like the `microsoft-graph`/`microsoft_graph` more. cc @fabpot